examples/animation/animate_decay.py

```diff
@@ -32,6 +32,6 @@ def run(data):
 
     return line,
 
-ani = animation.FuncAnimation(fig, run, data_gen, blit=True, interval=10,
+ani = animation.FuncAnimation(fig, run, data_gen, blit=False, interval=10,
```

Comment:
I thought that the animation code added hooks so that on a redraw event it takes care of re-copying the blit box not to resizes spcefically.  

What is going on on my system (I think) is that when you mouse in/out qt triggers a full re-paint (as there may have been silly alpha stuff going on when the window was non-focused (oh, right I also use focus follow mouse so I am really changing the focus state) ) and that repaint from the windowing system bubbles down and does a full re-draw of the canvas which updates the tick labels.